src/ArgumentCompleter.cs

```diff
@@ -20,16 +20,17 @@ internal class NameCompleter : IArgumentCompleter
 
     private IEnumerable<CompletionResult> CompleteRepositoryName(string wordToComplete)
     {
-        var res = new List<CompletionResult>();
+        List<CompletionResult> res = new List<CompletionResult>();

```

Comment:
This `var` makes sense: the type is explicit on the RHS
```suggestion
        var res = new List<CompletionResult>();

```